src/i18n/i18n.csv

```diff
@@ -163,6 +163,9 @@ Table enhancement,sortAsc,: activate for ascending sort,&#160;: activer pour tri
 ,info1000,",",&#160;,,&#160;,,,,",",",",&#160;,&#160;,&#160;,&#160;,&#160;,",",&#160;,,,&#160;,",",",",,,&#160;,&#160;,&#160;,,&#160;,.,",",,,
 ,lenMenu,Show _MENU_ entries,Afficher _MENU_ entrées,Wys _MENU_ inskrywings,Trego _MENU_ hyra,عرض _MENU_ مقالات,Ցուցադրել _MENU_ գրառում,Покажи _MENU_ вписвания,顯示_MENU_項,显示_MENU_项,Zobrazit _MENU_ položek,Toon _MENU_ lemma\'s,Näita _MENU_ kandeid,Zeige _MENU_ Einträge,Εμφάνιση _MENU_ καταχωρήσεις,_MENU_ प्रविष्टियों दिखाएं,Megjelenítés _MENU_ bejegyzés,Sýna _MENU_ færslur,Tampilkan _MENU_ entri,Mostra _MENU_ voci,_MENU_エントリーを表示,_MENU_ 항목을 표시,Parādīt _MENU_ ieraksti,Rodyti _MENU_ įrašai,Pokaż _MENU_ wpisów,Mostrar _MENU_ entradas,Mostrar _MENU_ entradas,Показать _MENU_ записей,Zobraziť _MENU_ položiek,Mostrar _MENU_ entradas,แสดงรายการ _MENU_,_MENU_ girdileri göster,Показати _MENU_ записів,Hiển thị _MENU_ mục
 ,filter,Filter items,Filtrer les articles,Filter artikels,Artikuj filtër,تصفية العناصر,Զտման նյութեր,Филтриране на елементи,過濾選項...,过滤项目,Filtrování položek,Artikelen filteren,Filtreeri,Filter Artikel,Φιλτράρισμα των στοιχείων,फ़िल्टर आइटम,Szűrés,Sía atriði,Filter artikel,Filtra gli articoli,検索条件を指定する,필터 항목,Atlasīt vienumu,Filtruoti,Filtruj pozycje,Filtrar os ítens,Filtrar os ítens,Фильтровать,Filtrovanie položiek,filtrar artículos,กรองรายการ,Maddeleri filtrele,Фільтр,Lọc mặt hàng
+,print,Print Table,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,open_window,(opens in a new window),,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,table_title,DataTable Print Window,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
These need to be added to the Google Spreadsheet or they'll be lost next time it's update. After the string are in the spreadsheet the `grunt update-i18n` task needs to be run.